.codeclimate.yml

```diff
@@ -62,9 +62,6 @@ plugins:
       - XXX
   pep8:
     enabled: true
-  rubocop:
```

Comment:
maybe we just remove all codeclimate like we did in the IDP?

Reply:
I think we still have code climate in the IDP? 
https://github.com/18F/identity-idp/blob/main/.codeclimate.yml